### DISCUSSION TITLE
is there a setting to make always  visible on desktop?

### DISCUSSION THREAD

**Discussion by jlitz** [Flow-Launcher/Flow.Launcher#1458] [General]
was wondering if always visible is an option rather than using hotkey

**Comment by taooceros** [comment]
I think there's a setting that won't hide the window when losing focus, which should do what you want i believe.

**Reply by jlitz** [reply]
yes I see that have it unchecked  but still stays hidden.

**Reply by taooceros** [reply]
![image](https://user-images.githubusercontent.com/45326534/194622425-1960b745-0651-48c0-8ee3-7826b58b6773.png)
As long as you have activated once, it will stay there until you activate again?
